community/kernel/src/main/java/org/neo4j/kernel/impl/store/MetaDataStore.java

```diff
@@ -808,7 +808,7 @@ public boolean closedTransactionIdIsOnParWithOpenedTransactionId()
         return onPar;
     }
 
-    public void logRecords( final Logger msgLog )
+    public void logRecords( final Consumer<String> consumer )
```

Comment:
No it is not that, it is that you don;t always want the output to be formatted like out log entries.
Specifically, you don't need a date and the log level